packages/blocks-ui/src/canvas.js

```diff
@@ -18,12 +22,57 @@ const Wrap = props => (
   />
 )
 
+function copyToClipboard(toCopy) {
+  const el = document.createElement(`textarea`);
+  el.value = toCopy;
+  el.setAttribute(`readonly`, ``);
+  el.style.position = `absolute`;
+  el.style.left = `-9999px`;
+  document.body.appendChild(el);
+  el.select();
+  document.execCommand(`copy`);
+  document.body.removeChild(el);
+}
+
+const Copy = ({ toCopy }) => {
+  const [hasCopied, setHasCopied] = useState(false);
+
+  function copyToClipboardOnClick() {
+    if (hasCopied) return;
+
+    copyToClipboard(toCopy);
+    setHasCopied(true);
+
+    setTimeout(() => {
+      setHasCopied(false);
+    }, 2000);
+  }
+
+  return (
+    <IconButton onClick={copyToClipboardOnClick} sx={{ float: "right" }}>
```

Comment:
Lets make the wrapping container `position: relative` so that we can change this styling to:

```js
position: 'absolute',
right: 0
```

This will ensure the button doesn't take up a full column on the right hand side which wouldn't be great for mobile.